src/Umbraco.Web.UI.Client/src/views/webhooks/overlays/edit.controller.js

```diff
@@ -28,23 +28,39 @@
       });
     }
 
-    function openContentTypePicker()
-    {
+    function openContentTypePicker() {
       const isContent = $scope.model.webhook ? $scope.model.webhook.events[0].toLowerCase().includes("content") : null;
-      editorService.treePicker({
-        section: 'settings',
-        treeAlias: isContent ? 'documentTypes' : 'mediaTypes',
-        entityType: isContent ? 'DocumentType' : 'MediaType',
+
+      const editor = {
         multiPicker: true,
+        filterCssClass: "not-allowed not-published",
+        filter: function (item) {
+          // filter out folders (containers), element types (for content) and already selected items
+          return item.nodeType === "container"; // || item.metaData.isElement || !!_.findWhere(vm.itemTypes, { udi: item.udi });
+        },
         submit(model) {
           getEntities(model.selection, isContent);
-          $scope.model.webhook.contentTypeKeys = model.selection.map((item) => item.key);
+          $scope.model.webhook.contentTypeKeys = model.selection.map(item => item.key);
           editorService.close();
         },
         close() {
           editorService.close();
         }
-      });
+      };
+
+      const itemType = isContent ? "content" : "media";
+
+      switch (itemType) {
+        case "content":
+          editorService.contentTypePicker(editor);
+          break;
+        case "media":
+          editorService.mediaTypePicker(editor);
+          break;
+        case "member":
+          editorService.memberTypePicker(editor);
+          break;
+      }
```

Comment:
It would be great if we could use this here: https://github.com/umbraco/Umbraco-CMS/pull/15080

Would it make sense the support webhooks for Member events @Zeegaan ?

Reply:
It is in the pipeline for rc-2 (but no promises 😛), so I think it's okay 👍

Reply:
At least it never opens member type picker for now and if https://github.com/umbraco/Umbraco-CMS/pull/15080 gets merged, we can eventually refactor this part, but anyway it should still work 😎